gcexport.py

```diff
@@ -1214,24 +1210,20 @@ def main(argv):
         if action == 's':
             # Display which entry we're skipping.
             print('Skipping   : Garmin Connect activity ', end='')
-            print('(', current_index, '/', len(action_list), ') ', sep='', end='')
-            print('[', actvty['activityId'], ']', sep='')
+            print(f"({current_index}/{len(action_list)}) [{actvty['activityId']}]")
             continue
 
         # Action: excluding
         if action == 'e':
             # Display which entry we're skipping.
             print('Excluding  : Garmin Connect activity ', end='')
-            print('(', current_index, '/', len(action_list), ') ', sep='', end='')
-            print('[', actvty['activityId'], '] ', sep='')
+            print(f"({current_index}/{len(action_list)}) [{actvty['activityId']}]")
             continue
 
         # Action: download
         # Display which entry we're working on.
         print('Downloading: Garmin Connect activity ', end='')
-        print('(', current_index, '/', len(action_list), ') ', sep='', end='')
```

Comment:
Are there some curly braces missing? When testing your branch my output was
`Downloading: Garmin Connect activity (1/x) [9007684183] actvty['activityName']`
instead of
`Downloading: Garmin Connect activity (1/x) [9007684183] My Activity Name`

Reply:
Good catch! Fixed in 3c62dc4.

PS. Looks like I ran the wrong version to test this branch. Tested now.